complete/src/main/java/com/example/restservice/dao/TokenRepository.java

```diff
@@ -2,16 +2,16 @@
 
 import com.example.restservice.constants.TokenStatus;
 import java.util.stream.Stream;
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 @EnableJpaRepositories
-public interface TokenRepository extends CrudRepository<Token, String> {
+public interface TokenRepository extends JpaRepository<Token, String> {
```

Comment:
Is there any advantage to making the token repository also extend jparepository? Or is this just to remain consistent?

Reply:
Ha consistency was my rationale